data/json/items/magazine/9mm.json

```diff
@@ -49,6 +49,86 @@
     "reliability": 8,
     "flags": [ "MAG_COMPACT" ]
   },
+  {
+    "id": "glock17_17",
+    "type": "MAGAZINE",
+    "name": "Glock 17 magazine - 17 rd",
+    "description": "Made for the Glock 17, ironically this magazine holds 17 rounds.",
+    "weight": 105,
+    "volume": 1,
+    "price": 2900,
+    "material": "plastic",
+    "symbol": "#",
+    "color": "ltgray",
+    "ammo_type": "9mm",
+    "capacity": 17,
+    "reliability": 7,
+    "reload_time": 140,
+    "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "glock17_22",
+    "type": "MAGAZINE",
+    "name": "Glock 17 magazine - 22 rd",
+    "description": "Made for the Glock 17, this magazine holds 22 rounds.",
+    "weight": 105,
+    "volume": 2,
+    "price": 3500,
+    "material": "plastic",
+    "symbol": "#",
+    "color": "ltgray",
+    "ammo_type": "9mm",
+    "capacity": 22,
+    "reliability": 8
+  },
+  {
+    "id": "glock17_33",
+    "type": "MAGAZINE",
+    "name": "Glock 17 magazine - 33 rd",
```

Comment:
The Glock extended mag is 30-rounds already so having both a 30 and 33 round mag is kinda overkill. Either one or the other by which one is more commercially available would be better I think.

Reply:
The 30 round magazine seems to be more popular, removing the 30 round.